noah/inline_img.css

```diff
@@ -0,0 +1,25 @@
+            /*correct way to inline image*/
+            /*put this class on the div containing the paragraph*/
+            .paragraph_container{
+            height:480px;
+            width:480px;
+            border-style:dashed;
+            display:inline-block;
+            vertical-align: middle;
+            }
+            .paragraph_container > div{
+            	display:inline-block;
+            	vertical-align: middle;
+
+            }
+
+            .paragraph_container > p{
+            	position:relative;
+            }
+            /*a div with this class containing an image can be nested in a <p> */
+            .inline_image_right{
+            	float:right;
+            }
+            .inline_image_left{
```

Comment:
To keep images from being right against text, there should be: `padding: 10px 10px 10px 0;`.